src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,49 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '!!raw-loader!../wand_icon.svg';
+
+document.addEventListener('wagtail-ai:image-form', (event) => {
+  const input = event.target as HTMLInputElement;
+  const imageId = input.dataset['wagtailai-image-id']!;
+  const csrfToken = (
+    input.form!.querySelector(
```

Comment:
**Question:** We could test for the existence of form using an if statement and raising an explicit exception with a clear error, instead of using a non-null assertion operator.

Reply:
It seems such a niche error case though. If indeed the form is missing, one would have to dive into the code and debug it anyway, so I'm not sure that a clear error message would help.

Reply:
Eh, I thought the non-null assertion would insert a check, and throw an error, but it does no such thing. I've added a check as you suggested.